pkg/controller/controller.go

```diff
@@ -522,3 +522,27 @@ func isInstanceReady(instance *v1alpha1.Instance) bool {
 
 	return false
 }
+
+func isOwnedByBinding(objectMeta *metav1.ObjectMeta, binding *v1alpha1.Binding) bool {
+	ref := getControllerRef(objectMeta)
+	if ref == nil {
+		return false
+	}
+	return ref.APIVersion == binding.APIVersion &&
+		ref.Kind == binding.Kind &&
+		ref.Name == binding.Name &&
```

Comment:
@pmorie the docs do have all the fields set: https://kubernetes.io/docs/concepts/workloads/controllers/garbage-collection/#owners-and-dependents
I can also check some Kube source code as an example of using ownerReferences